test/RuntimeInformationTest.cpp

```diff
@@ -21,13 +21,14 @@ TEST(RuntimeInformation, addLimitOffsetRow) {
 
   ASSERT_EQ(rti.children_.size(), 1u);
   const auto& child = rti.children_.at(0);
-  EXPECT_EQ(child.descriptor_, "BaseOperation");
-  EXPECT_EQ(child.totalTime_, 4.0);
-  EXPECT_EQ(child.getOperationTime(), 4.0);
-  EXPECT_TRUE(child.details_.at("not-written-to-cache-because-child-of-limit"));
+  EXPECT_EQ(child->descriptor_, "BaseOperation");
```

Comment:
Better: dereference the pointer and store it, then the diff gets much smaller and more visible (across this whole test).